### PR TITLE
Improve login page design

### PR DESCRIPTION
**Pull request by jessieay**
* this page may be removed soon, but I already put together new design
 so may as well commit?
* closes https://github.com/18F/micropurchase/issues/776

screen shot 2016-07-06 at 2 21 22 pm

### CODE REVIEW

File: app/views/components/_guest_nav_drawer.html.erb [hunk 1]
```diff
@@ -1,2 +1,3 @@
-<!-- <a href="/login" class="button&#45;login usa&#45;button usa&#45;button&#45;outline&#45;inverse">Login</a> -->
-<%= link_to 'Login', login_path, class: 'usa-button usa-button-outline-inverse' %>
+<% if request.path != '/login' %>
```
Comment (adelevie) [inline app/views/components/_guest_nav_drawer.html.erb]: very logic-y for a view? I guess if the thing will get tossed out, then no big deal?

**Comment by adelevie** [comment]
one comment, otherwise LGTM

File: app/views/components/_guest_nav_drawer.html.erb (hunk 1 above)
Comment (jessieay) [inline app/views/components/_guest_nav_drawer.html.erb]: yeah I wasn't sure how else to do it. will probably be thrown out but I am open to ideas on a better way!

Comment (adelevie) [inline app/views/components/_guest_nav_drawer.html.erb]: controller:
```ruby
@request_presenter = RequestPresenter.new(request)
```

view:
```erb
<% if @request_presenter.show_login_button? %>
```

I can't tell if I'm joking or not.

Comment (jessieay) [inline app/views/components/_guest_nav_drawer.html.erb]: haha that's still a conditional in the view!

Comment (adelevie) [inline app/views/components/_guest_nav_drawer.html.erb]:
```erb
<%= render partial: @request_presenter.login_button_partial %>
```

:trollface:

**Pull request merged**